core/src/OpenMP/Kokkos_OpenMP_Instance.cpp

```diff
@@ -55,19 +55,17 @@ void OpenMPInternal::clear_thread_data() {
 
   OpenMP::memory_space space;
 
-#pragma omp parallel num_threads(m_pool_size)
-  {
-    const int rank = omp_get_thread_num();
-
+  for (int rank = 0; rank < m_pool_size; ++rank) {
```

Comment:
It would seg. fault for a few `Debug` configurations (see report attached in https://github.com/kokkos/kokkos/pull/7600#issuecomment-2541834725). The seg. fault would disappear if I remove the `omp parallel` region.

Reply:
Per dev meeting, the parallel region is not essential.